cfme/tests/test_db_migrate.py

```diff
@@ -64,6 +85,95 @@ def test_db_migrate(app_creds, temp_appliance_extended_db, db_url, db_version, d
     # Download the database
     logger.info("Downloading database: {}".format(db_desc))
     url_basename = os_path.basename(db_url)
+    # MBU Backup is to large for /tmp
+    loc = "/tmp/" if db_desc != "MBU Backup" else "/"
+    rc, out = app.ssh_client.run_command(
+        'curl -o "{}{}" "{}"'.format(loc, url_basename, db_url), timeout=30)
+    assert rc == 0, "Failed to download database: {}".format(out)
+
+    # The v2_key is potentially here
+    v2key_url = os_path.join(os_path.dirname(db_url), "v2_key")
+
+    # Stop EVM service and drop vmdb_production DB
+    app.evmserverd.stop()
+    app.db.drop()
+
+    # restore new DB and migrate it
+    with app.ssh_client as ssh:
```

Comment:
**REQUIRED**
This context manager is not needed.

Reply:
Understood, this will be changed in my second re-factoring PR #6349